include/nvector/nvector_parallel.h

```diff
@@ -122,7 +122,7 @@ SUNDIALS_EXPORT void N_VSpace_Parallel(N_Vector v, sunindextype *lrw,
                                        sunindextype *liw);
 SUNDIALS_EXPORT sunrealtype *N_VGetArrayPointer_Parallel(N_Vector v);
 SUNDIALS_EXPORT void N_VSetArrayPointer_Parallel(sunrealtype *v_data, N_Vector v);
-SUNDIALS_EXPORT void *N_VGetCommunicator_Parallel(N_Vector v);
+SUNDIALS_EXPORT MPI_Comm N_VGetCommunicator_Parallel(N_Vector v);
```

Comment:
Should this be `SUNComm`?
```suggestion
SUNDIALS_EXPORT SUNComm N_VGetCommunicator_Parallel(N_Vector v);
```

Reply:
No, its an MPI only vector, there is no reason to use a SUNComm here (which would just be a typedef to MPI_Comm anyways). I guess we could, but I think I slightly prefer using MPI_Comm here.

Reply:
That's why I had the "?" in my comment.  I'm good with keeping this as-is.